scripts/release_tools/import_prod.sh

```diff
@@ -9,23 +9,25 @@
 #   * The currently logged in gcloud credentials have access to the broad-dsp-monster-hca-prod google project
 #
 # Invoking the script:
-# ./import_prod.sh <bucket name> <subdir of bucket where the staging area resides>
+# ./import_prod.sh <bucket name> <subdir of bucket where the staging area resides> <staging bucket prefix>
 #
 # Example:
-# ./import_prod.sh broad-foo-bar-bucket-name /prod/baz-staging-area
+# ./import_prod.sh broad-foo-bar-bucket-name /prod/baz-staging-area dcp2
 
 SOURCE_BUCKET_NAME=$1
 SOURCE_BUCKET_PREFIX=$2
 
-STAGING_BUCKET=dcp2_$(gdate +%Y_%m_%d_%H%M%S)
+STAGING_BUCKET_PREFIX=$3
+STAGING_BUCKET=${STAGING_BUCKET_PREFIX}_$(gdate +%Y_%m_%d_%H%M%S)
 
 # always run these imports in our prod k8s cluster
 CURRENT_CLUSTER=$(kubectl config current-context)
-if [ $CURRENT_CLUSTER != "gke_mystical-slate-284720_us-central1-c_hca-cluster" ] ; then
-  echo "Connecting to HCA prod cluster..."
+if [[ $CURRENT_CLUSTER != "gke_mystical-slate-284720_us-central1-c_hca-cluster" ]] ; then
+  echo "Connecting to HCA  prod cluster..."
   gcloud container clusters get-credentials hca-cluster --zone us-central1-c --project mystical-slate-284720
 fi
 
```

Comment:
I like the idea of echoing this for diagnostic purposes, can you add a bit more context to this message?